### PR TITLE
Add example of using File.expand_path

### PR DESCRIPTION
**Pull request by tannerwelsh**
Show how to create absolute pathnames from paths relative to `__FILE__`.

### CODE REVIEW

**Comment by tannerwelsh** [comment]
@zspencer & @jfarmer any thoughts on how to better explain / provide examples for `File.expand_path`?

**Comment by zspencer** [comment]
I'd also demonstrate `File.expand_path('~/')`

It'd be also useful to demonstrate that `File.expand_path('.')` / `File.expand_path('..')` is different depending n where you ran the command from.

**Comment by tannerwelsh** [comment]
Thanks, @zspencer . Added a few changes based on your comment.

File: examples/expand_path.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+puts "The absolute path of this Ruby file is:"
+puts "=> #{File.expand_path(__FILE__)}"
+
+puts ""
+puts "The relative path from this file to `args.rb` is:"
+puts "=> #{"./args.rb"}"
+
+puts ""
+puts "Does `File.exists?` return true for the relative path `./args.rb`?"
+puts "=> #{File.exists?("./args.rb")}"
```
Comment (jfarmer) [inline examples/expand_path.rb]: `File.exists?` is deprecated in favor of `File.exist?`.  The Ruby interpreter will spit out a warning if warnings are enabled.

**Comment by jfarmer** [comment]
Modulo my comment about `File.exists?`, it looks good!

**Comment by tannerwelsh** [comment]
Thanks @jfarmer! Merging.

**Pull request merged**